product_docs/docs/biganimal/release/free_trial/index.mdx

```diff
@@ -7,36 +7,11 @@ navigation:
-The free trial provides a subset of the full-featured BigAnimal environment so that you can get a feel for what to expect. 
+The BigAnimal free trial gives you $300 worth of free credits to use EDB's cloud account to experience the features and capabilities of BigAnimal.
 
 !!! Note "Get started with BigAnimal quickly"
     [Quick start for the BigAnimal free trial](quickstart) &mdash; Create, connect to, and work with 
-      a standard community PostgreSQL 14 cluster, provisioned on AWS. 
-
-## What you get as part of the free trial
-
-- **The latest & greatest PostgreSQL and EDB Postgres Advanced Server databases.** 
-  - [PostgreSQL 14](https://www.enterprisedb.com/blog/cool-new-contributions-postgresql-14) is more flexible, more secure, and faster than ever ([release notes](https://www.postgresql.org/docs/14/release.html)). Or, try out BigAnimal's support of PostgresSQL 15 ([release notes](https://www.postgresql.org/docs/15/release.html)).
-
-  - [EDB Postgres Advanced Server 14](https://www.enterprisedb.com/docs/epas/latest/epas_rel_notes/) builds on PostgreSQL 14 with lots of features for Oracle database compatibility, security, and usability ([more on Oracle database compatibility](/epas/latest/epas_compat_ora_dev_guide/)).
-- **One active cluster, unlimited inactive ones.** 
-  - You can try different configurations by deleting the active cluster and creating a new one.
-  - Restore deleted clusters at any time within the trial period.
-  - Choose from either AWS or Azure cloud hosting, with two regions available for each (out of [all supported regions](/biganimal/latest/overview/03a_region_support)).
-  
-    | Cluster details | On Azure                                      | On AWS                                                |
-    | --------------- | --------------------------------------------- | ----------------------------------------------------- |
-    | Region          | Either US East 2 or West Europe (Netherlands) | Either US-East 1 (N. Virginia) or EU-West-1 (Ireland) |
-    | Instance Type   | Esv3 series 2 vCPU with 16 GB RAM             | r5.large                                              |
-    | Storage         | Premium SSD disks P1 4Gi                      | GP3 Volume disk size 4                           |
-    |                 |                                               |                                                       |
-- **Edit database configuration settings.**
-- **Connect to your cluster** by way of a public IP from psql, pgAdmin, or your favorite PostgreSQL-friendly application.
-- **Access to EDB enterprise-level support.**
-- **Use BigAnimal services for 14 days.** The countdown begins when you first provision a cluster. 
-
-Numerous other instance types, storage types, regions, customizations, and support options are available with a paid account.
+      a standard community PostgreSQL 15 cluster, provisioned on AWS. 
```

Comment:
pardon my ignorance here, but are we saying customers are only allowed to provision PostgreSQL 15 during the use of their free credits?  or is that simply a recommendation here? @drothery-edb